app/src/test/java/protect/card_locker/ImportURITest.java

```diff
@@ -70,7 +70,7 @@ public void ensureNoDataLoss() throws InvalidObjectException, UnsupportedEncodin
     @Test
     public void ensureNoCrashOnMissingHeaderFields() throws InvalidObjectException, UnsupportedEncodingException {
         // Generate card
-        DBHelper.insertLoyaltyCard(mDatabase, "store", "note", null, new BigDecimal("10.00"), Currency.getInstance("EUR"), BarcodeFormat.UPC_A.toString(), null, CatimaBarcode.fromBarcode(BarcodeFormat.QR_CODE), null, 0, null);
+        DBHelper.insertLoyaltyCard(mDatabase, "store", "note", null, new BigDecimal("10.00"), Currency.getInstance("EUR"), BarcodeFormat.UPC_A.toString(), null, CatimaBarcode.fromBarcode(BarcodeFormat.QR_CODE), null, 0, null,0);
```

Comment:
Please also test if the value is correct in // Compare everything

In this case it should be 0 because if someone shares a card with us, there is no data if that user had it archived.